pkgs/applications/editors/vim/macvim.nix

```diff
@@ -61,7 +61,7 @@ stdenv.mkDerivation {
       "--without-local-dir"
       "--with-luajit"
       "--with-lua-prefix=${luajit}"
-      "--with-python3-command=${python37}/bin/python3"
+      "--with-python3-command=${python3}/bin/python3"
```

Comment:
note this can be `python3.interpreter`. That way it can even be substituted with say pypy.